pkgs/servers/monitoring/mtail/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, gotools, buildGoPackage }:
+
+buildGoPackage rec {
+  name = "mtail-${version}";
+  version = "v3.0.0-rc4";
```

Comment:
`v` should not be a part of the version, it is only a part of `rev`.

Use commit message `mtail: init at 3.0.0-rc4` (see https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).